.travis.yml

```diff
@@ -1,9 +1,8 @@
 sudo: false
 language: node_js
 node_js:
+  - '8'
```

Comment:
FYI, I opened PR #468 to to test against Node.js v8, which can be closed if this is merged.